monkey/infection_monkey/ransomware/ransomware_payload.py

```diff
@@ -17,7 +17,10 @@
 
 class RansomewarePayload:
     def __init__(self, config: dict, telemetry_messenger: ITelemetryMessenger):
-        target_directories = config["directories"]
+        self.should_encrypt = config["encryption"]["should_encrypt"]
```

Comment:
`should` is kind of confusing. Maybe `to_encrypt` or not, the question is now :sweat_smile:

Reply:
What about "enabled" ?